tools/wds2idx.py

```diff
@@ -100,7 +100,7 @@ def _get_data_tar(self):
         )  # <type>... <size> <date> <name>
 
         # Extracting
-        for blocks_line, next_blocks_line, types_sizes_line in zip(tar_blocks, tar_blocks[1:],
+        for blocks_line, next_blocks_line, types_sizes_line in zip(tar_blocks, tar_blocks,
```

Comment:
Do we need to zip the same thing twice? 
We should be using current blocks_line in the calculation I guess.

Reply:
Fixed